src/packages/@ncigdc/tableModels/projectsTableModel.js

```diff
@@ -103,21 +105,20 @@ const projectsTableModel = [
     downloadable: true,
     th: () => <Th rowSpan="2">Primary Site</Th>,
     td: ({ node }) =>
-      <Td key="primary_site">
-        <CollapsibleList
-          data={sortBy(node.primary_site)}
-          limit={node.primary_site.length > 1 ? 0 : 1}
-          {...(node.primary_site.length === 1
-            ? {}
-            : {
-                AlternateText: () => <span>Multiple Primary Sites</span>,
-                ToggleText: ({ expanded }) =>
-                  expanded
-                    ? <span>{`\u25B4 less`}</span>
-                    : <span>{`\u25BE Show ${node.primary_site
-                        .length} items`}</span>,
-              })}
-        />
+      <Td
+        key="primary_site"
+        style={{ maxWidth: '200px', padding: '3px 15px 3px 3px' }}
```

Comment:
hmm, the text isn't wrapping, do you need whitespace normal here?

Reply:
<img width="287" alt="screen shot 2017-09-28 at 2 28 17 pm" src="https://user-images.githubusercontent.com/1314446/30983760-526355cc-a459-11e7-8d3a-811488d9a48e.png">
isn't it? I'm looking at the wrong spot?

Reply:
oh hmmm, maybe I didn't have the up to date code, I will test again!

Reply:
Ahh I see, I am looking at a single value, so it's not in the collapsed list. so the single value needs it I think?
![image](https://user-images.githubusercontent.com/3953093/31017040-29ff3668-a4f5-11e7-9b4f-c05aeeeb0465.png)

Reply:
ohhhh

Reply:
fixed, thank you 😊